lib/collection/proxy-config.js

```diff
@@ -1,26 +1,33 @@
 var _ = require('../util').lodash,
     Property = require('./property').Property,
+    url = require('url'),
```

Comment:
This should be referenced with a sufficiently unique name to avoid confusion between `Url` and `url`. Something like:
```js
nodeUrl = require('url'),
Url = require('./url').Url;
```